core/importer/datasets_importer.py

```diff
@@ -15,7 +15,7 @@ class DatasetsImporter(BaseImporter):
     """
 
     json_schema_validator = DatasetJSONSchemaValidator()
-    json_schema_uri = 'https://raw.githubusercontent.com/elixir-luxembourg/json-schemas/master/schemas/elu-dataset.json'
+    #json_schema_uri = 'https://raw.githubusercontent.com/elixir-luxembourg/json-schemas/master/schemas/elu-dataset.json'
```

Comment:
@jLebioda what do you think? Can I include the [repository with JSON schemas](https://github.com/elixir-luxembourg/json-schemas) as a submodule? It makes more sense than having one url which might not work and another fallback copy of schemas in fixtures.